[Bookings] Add checkmark to attendance status sheet

irfano opened the pull request:
<!-- Remember about a good descriptive title. -->

<!-- Id number of the Linear issue this PR addresses, e.g., WOOMOB-373. -->
Closes WOOMOB-1606

### Description
<!-- Take the time to write a good summary. Why is it needed? What does it do? When fixing bugs try to avoid just writing “See original issue” – clarify what the problem was and how you’ve fixed it. -->
This adds a checkmark icon to the selected attendance stat on the sheet. 

### Test Steps
<!-- Describe how to test your changes. Include only what’s needed for the reviewer to validate the behavior.
For new features, outline the main user flow or goal rather than every tap or click — the reviewer should be able to complete the flow naturally.
If applicable, mention key devices, scenarios, or edge cases to verify. -->
1. Log in with a CIAB store.
2. Go to Bookings.
3. Select a booking.
4. Tap ATTENDANCE Status row.
5. Change the selected status.

### Images/gif
<!-- Include before and after images or gifs when appropriate. -->
|Before|After|After dark|
|-|-|-|
|<img width="360" src="https://github.com/user-attachments/assets/dc3ca4d3-a0b3-4cca-a631-20745c1db46d" />|<img width="360" src="https://github.com/user-attachments/assets/719b27f1-b381-4079-9797-1394527f3c4e" />|<img width="360" src="https://github.com/user-attachments/assets/34053808-38f1-4fe0-a6a3-99d088c61ef8" />|

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

<!-- Pull request guidelines: https://github.com/woocommerce/woocommerce-android/blob/develop/docs/pull-request-guidelines.md -->

irfano committed:
Show a checkmark for the selected attendance status

Adds a checkmark icon to the currently selected item in the booking attendance status bottom sheet.

irfano committed:
Display selected attendance status on attendance status sheet

Pass the current booking attendance status to the `BookingAttendanceStatusBottomSheet` to ensure the correct status is selected when the sheet is opened.

AdamGrzybkowski commented on WooCommerce/src/main/kotlin/com/woocommerce/android/ui/bookings/compose/BookingAttendanceStatusBottomSheet.kt:
```diff
@@ -117,6 +123,15 @@ private fun AttendanceStatusRow(
+            if (isSelected) {
+                Icon(
+                    painter = painterResource(id = R.drawable.ic_done_secondary),
+                    contentDescription = null,
```
It would make sense to add the contentDescription here, since the information about the selected value is quite important.

irfano replied:
Good point! Added: 6943e1399b383ec2ac24ce87f835350e6df2a523

AdamGrzybkowski reviewed (approved):
Looks good! Left one comment worth addressing.

irfano committed:
Add content description for selected booking filter

Adds a content description to the "selected" checkmark icon in the booking attendance status filter bottom sheet to improve accessibility.
